### PR TITLE
feature: create an authentification pages

### PR DESCRIPTION
**Pull request by manzcode**
Create the authentification pages interface

### CODE REVIEW

**Review by marktlinn — COMMENTED** [review]
Code looks nice Mario 👌🏻. I like the conditional rendering of the signup and login form. Do you think we'll need different routes for the signup and login, as sometimes user's will want to go directly from the landing page to the signup, or could it be handled with props?

File: client/src/pages/auth/Auth.jsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                  <TfiFacebook
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded 
+              drop-shadow-lg hover:fill-white hover:bg-grotto-100"
+                    onClick={facebookAuth}
+                  />
+                  <BsLinkedin
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded
```
Comment (marktlinn) [inline client/src/pages/auth/Auth.jsx]: maybe border-ivory-300 works better with the linked auth's border's as it's the same colour as the hr's above?

File: client/src/pages/auth/Auth.jsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded 
+              drop-shadow-lg hover:fill-white hover:bg-grotto-100"
+                    onClick={googleAuth}
+                  />
+                  <TfiFacebook
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded
```
Comment (marktlinn) [inline client/src/pages/auth/Auth.jsx]: maybe border-ivory-300 works better with the linked auth's border's as it's the same colour as the hr's above?

File: client/src/pages/auth/Auth.jsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                <hr className="border-ivory-300 surlignage" />
+              </div>
+              <div className={`${!isSignUp && "md:h-12"}`}></div>
+              <br />
+              <div className="flex justify-center">
+                <div className="flex flex-row justify-between w-3/4">
+                  <BsGoogle
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded
```
Comment (marktlinn) [inline client/src/pages/auth/Auth.jsx]: maybe border-ivory-300 works better with the linked auth's border's as it's the same colour as the hr's above?

**Comment by manzcode** [comment]
if we're using redux, I would just check the status to render the form as sign up

**Comment by marktlinn** [comment]
> if we're using redux, I would just check the status to render the form as sign up

OK, Awesome. Nice one, Mario!

**Review by marktlinn — APPROVED** [review]
Great work! 😊

**Pull request merged**